src/Setup/Microsoft.VisualStudio.Templates.CS.EdgeDriverTestCore.vsmanproj

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

Reply:
> 
> 
> What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

One is for .Net Framework, one for .Net Core

Reply:
> 
> 
> What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

I have checked with Manish, for those two templates we just want to ship it with VS, currently not for CLI